install/create-docker-volumes.sh

```diff
@@ -1,10 +1,21 @@
 echo "${_group}Creating volumes for persistent storage ..."
 
-echo "Created $(docker volume create --name=sentry-clickhouse)."
-echo "Created $(docker volume create --name=sentry-data)."
-echo "Created $(docker volume create --name=sentry-kafka)."
-echo "Created $(docker volume create --name=sentry-postgres)."
-echo "Created $(docker volume create --name=sentry-redis)."
-echo "Created $(docker volume create --name=sentry-symbolicator)."
+create_volume() {
+  create_command="$CONTAINER_ENGINE volume create"
+  if [ "$CONTAINER_ENGINE" = "podman" ]; then
+    create_command="$create_command --ignore $1"
+  else
+    create_command="$create_command --name=$1"
+  fi
+
+  $create_command
+}
```

Comment:
`--ignore` is unknown flag to podman

```suggestion
  declare -a create_command=("$CONTAINER_ENGINE" volume create)
  if [ "$CONTAINER_ENGINE" = "podman" ]; then
    create_command+=("$1")
  else
    create_command+=("--name=$1")
  fi

  "${create_command[@]}"
}
```